generators/app/index.js

```diff
@@ -404,6 +413,55 @@ module.exports = generators.Base.extend({
           }
         }
       }.bind(self));
+    },
+
+    addGitignore: function (self, cb) {
+      self = self || this;
+      cb = cb || this.async();
+      var cssCompiledPath = self.staticsRoot + '/' + (self.cssCompiledOutput).split('/')[0],
+          jsCompiledPath = self.staticsRoot + '/' + (self.jsCompiledOutput).split('/')[0],
+          whenIgnoring = function () {
+            return self.gitIgnore;
+          };
+
+      return self.prompt(
+        [
+          {
+            when: whenIgnoring,
+            type: 'input',
+            name: 'gitIgnoreStream',
+            message: 'What location would you like you add to .gitignore?',
```

Comment:
Even simpler, just toss it into the options pile. That way it can be taken as an argument like all the other features which are considered too niche to prompt for.